### DISCUSSION TITLE
on_tick没被调用

### DISCUSSION THREAD

**Discussion by sichal** [wondertrader/wondertrader#43] [其他/General]
很可能是on_init里没有stra_sub_tick订阅行情，我就犯过这个错误，一不小心把订阅函数删了

**Comment by ZzzzHeJ** [comment]
建议按照如下流程排查：

1. 检查回测时间是否正确
2. 检查回测时间中是否有tick数据
3. 检查是否订阅tick数据
4. 检查订阅代码是否正确

**Reply by wondertrader** [reply]
on_tick改成需要显式订阅了，可以尝试在on_init里调用context.stra_sub_ticks('SHFE.rb.2205')
